client/signup/steps/creds-complete/index.jsx

```diff
@@ -21,25 +22,30 @@ class CredsCompleteStep extends Component {
 		positionInFlow: PropTypes.number,
 		signupProgress: PropTypes.array,
 		stepName: PropTypes.string,
+		signupDependencies: PropTypes.object,
 	};
 
 	renderStepContent = () => {
-		const { translate, wpAdminUrl } = this.props;
+		const { translate, wpAdminUrl, signupDependencies } = this.props;
 
 		return (
 			<Card className="creds-complete__card">
 				<h3 className="creds-complete__title">{ translate( 'Your site is set up and ready!' ) }</h3>
```

Comment:
This headline is a little weird in the context of adding credentials from the activity log. Would it be easy to change for just that flow? Perhaps the person could just exit the flow from the second skip link instead?

Reply:
@MichaelArestad yes, it's possible, and exiting through the second link would've required less refactoring. Please confirm if we'll go with that and I'll adapt things.

Reply:
Let's exit through the second skip link for the JITM/Banner flow.